src/utils/index.js

```diff
@@ -9,6 +9,7 @@ module.exports.checkHeadsetConnected = require('./checkHeadsetConnected');
 module.exports.debug = require('./debug');
 module.exports.entity = require('./entity');
 module.exports.forceCanvasResizeSafariMobile = require('./forceCanvasResizeSafariMobile');
+module.exports.functions = require('./functions');
```

Comment:
Agreed @ngokevin.  We could add `forEach` (in some places we're doing `[].forEach` and others we're doing `Array.prototype.forEach(NodeList)`, etc.  I'm sure it will grow.